File: courseproject-fitnessapp-asp/ClientApp/src/app/models/account.ts
import { Guid } from "guid-typescript";

export class Account {
  constructor(
   // public id?: Guid,
    public id?: string,
    public username?: string,
    public password?: string,
    public role?: Role,
    public email?: string,
    public confirmed?: boolean
  ) { }
}
export enum Role {
  User,
  Admin
}

File: courseproject-fitnessapp-asp/Data/Models/Account.cs
namespace courseproject_fitnessapp_asp.Data
{
    public class Account
    {
       // public Guid id { get; set; }
        public string id { get; set; }
        public string username { get; set; }
        public string password { get; set; }
        public Role role { get; set; }
        public string email { get; set; }
        public bool confirmed { get; set; }
    }

    public enum Role
    {
        User,
        Admin
    }
}
